src/ResourceManager/Resources/Commands.Resources/ManagementPartner/NewManagementPartner.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System.Management.Automation;
+using Microsoft.Azure.Management.ManagementPartner;
+using PartnerResources = Microsoft.Azure.Commands.Resources.Properties.Resources;
+
+
+namespace Microsoft.Azure.Commands.Resources
+{
+    [Cmdlet(VerbsCommon.New, "AzureRmManagementPartner", SupportsShouldProcess = true), OutputType(typeof(PSManagementPartner))]
+    public class NewManagementPartner:AzureManagementPartnerCmdletsBase
+    {
+        [Parameter(Position = 0, Mandatory = true, HelpMessage = "The management partner id")]
+        [ValidateNotNull]
+        public string PartnerId { get; set; }
+
+        public override void ExecuteCmdlet()
+        {
+            if (ShouldProcess(string.Format(PartnerResources.NewManagementParnterTarget, PartnerId),
+                string.Format(PartnerResources.NewManagementParnterAction,PartnerId)))
+            {
+                var response = new PSManagementPartner(AceProvisioningManagementPartnerApiClient.Partner
```

Comment:
If this is a long running operation (more than 10 seconds), please implement [-AsJob](https://github.com/Azure/azure-powershell/wiki/PowerShell-Cmdlet-Design-Guidelines#asjob-parameter). If not, this is fine.

Reply:
this is not a long running operation, usually take less than 1 second.